app/models/dispatch.rb

```diff
@@ -24,9 +24,15 @@
 #
 
 class Dispatch < ApplicationRecord
+    include Postable 
+
     acts_as_paranoid 
 
     belongs_to :fdp 
     
     has_many :dispatch_items
+
+    after_save :pre_post
+
+    after_update :reverse
```

Comment:
I actually was going to change the :prepost to :post  but i have another post method which actually does the posting. that'll soon be fixed.